### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by RuslanIbIsh**

### CODE REVIEW

**Review by Ilya-brinchuk — APPROVED** [review]

File: src/main/java/com/iri/movietickets/Main.java [hunk 1]
```diff
@@ -59,5 +60,9 @@ public static void main(String[] args) {
         shoppingCartService.addSession(movieSession, userTwo);
         ShoppingCart shoppingCart = shoppingCartService.getByUser(userTwo);
         shoppingCartService.clear(shoppingCart);
+
+        OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
+        orderService.completeOrder(shoppingCart);
```
Comment (KseniiaMakarova) [inline src/main/java/com/iri/movietickets/Main.java:65]: let's fill the cart before completing order

File: src/main/java/com/iri/movietickets/Main.java [hunk 2]
```diff
@@ -59,5 +60,9 @@ public static void main(String[] args) {
...
+        orderService.getOrderHistory(userOne);
```
Comment (KseniiaMakarova) [inline src/main/java/com/iri/movietickets/Main.java:66]: let's print out the result and see if everything is printed out ok

File: src/main/java/com/iri/movietickets/service/impl/OrderServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        Order order = new Order();
+        order.setTickets(new ArrayList<>(shoppingCart.getTickets()));
```
Comment (KseniiaMakarova) [inline src/main/java/com/iri/movietickets/service/impl/OrderServiceImpl.java:24]: what other fields does order have that should be filled here?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/iri/movietickets/Main.java (hunk 1 above)
Comment (RuslanIbIsh) [reply src/main/java/com/iri/movietickets/Main.java]: done

File: src/main/java/com/iri/movietickets/Main.java (hunk 2 above)
Comment (RuslanIbIsh) [reply src/main/java/com/iri/movietickets/Main.java]: done

File: src/main/java/com/iri/movietickets/service/impl/OrderServiceImpl.java (hunk 3 above)
Comment (RuslanIbIsh) [reply src/main/java/com/iri/movietickets/service/impl/OrderServiceImpl.java:25]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**